test/dataframes.jl

```diff
@@ -123,6 +123,54 @@ end
     @test d ≅ @transform(df, n1 = :i .* :g, n2 = :i .* :g)
 end
 
+@testset "@transform with @byrow" begin
+    df = DataFrame(
+        g = [1, 1, 1, 2, 2],
```

Comment:
Keep on previous line? Same below.

Reply:
Will clean up along with other tests after this.